external/java/src/main/java/ch/epfl/dedis/lib/SkipBlock.java

```diff
@@ -94,6 +103,42 @@ public int getIndex(){
         return ret;
     }
 
+    /**
+     * This function checks whether all signatures in the forward-links
+     * are correctly signed by the aggregate public key of the roster
+     *
+     * @return true if the signature is ok.
+     */
+    public boolean verifyForwardSignatures() {
+        List<Point> publics;
+        try {
+            publics = new Roster(this.skipBlock.getRoster()).getServicePublics("Skipchain");
```

Comment:
Do we have a constant or something for the "Skipchain" service name? Just in case the service name changes.